src/components/ObsDetails/DetailsTab.js

```diff
@@ -0,0 +1,157 @@
+// @flow
+
+import {
+  Body2,
+  Body4,
+  Button,
+  DateDisplay,
+  Divider,
+  Heading4,
+  ObservationLocation,
+  QualityGradeStatus
+} from "components/SharedComponents";
+import KebabMenu from "components/SharedComponents/KebabMenu";
+import Map from "components/SharedComponents/Map";
+import { View } from "components/styledComponents";
+import { t } from "i18next";
+import _ from "lodash";
+import type { Node } from "react";
+import React, { useState } from "react";
+import { Menu } from "react-native-paper";
+import IconMaterial from "react-native-vector-icons/MaterialIcons";
+import useIsConnected from "sharedHooks/useIsConnected";
+
+import Attribution from "./Attribution";
+import checkCamelAndSnakeCase from "./helpers/checkCamelAndSnakeCase";
+
+type Props = {
+  observation: Object
+}
+
+const headingClass = "mt-[20px] mb-[11px] text-black";
+const sectionClass = "mx-[15px] mb-[20px]";
+
+const DetailsTab = ( { observation }: Props ): Node => {
+  const isOnline = useIsConnected( );
+  const application = observation?.application?.name;
+  const [locationKebabMenuVisible, setLocationKebabMenuVisible] = useState( false );
+
+  const displayQualityGradeOption = option => {
+    const qualityGrade = observation?.quality_grade;
+    const labelClassName = ( qualityGrade === option ) ? "font-bold" : "";
+
+    return (
+      <View className="flex-col space-y-[8px]">
+        <QualityGradeStatus qualityGrade={option} />
+        <Body4 className={labelClassName}>{_.startCase( _.camelCase( t( option ) ) )}</Body4>
+      </View>
+    );
+  };
+
+  return (
+    <>
+      {observation.description && (
+        <>
+          <View className={sectionClass}>
+            <Heading4 className={headingClass}>{t( "NOTES" )}</Heading4>
+            <Body2>{observation.description}</Body2>
+          </View>
+          <Divider />
+        </>
+      )}
+
+      <View className="flex-row justify-between items-center mt-[8px] mx-[15px]">
+        <Heading4>{t( "LOCATION" )}</Heading4>
+        <KebabMenu
+          visible={locationKebabMenuVisible}
+          setVisible={setLocationKebabMenuVisible}
+        >
+          <Menu.Item
+            onPress={async ( ) => {
+
+            }}
+            title={t( "Share-location" )}
+          />
+          <Menu.Item
+            onPress={async ( ) => {
+
+            }}
+            title={t( "Copy-coordinates" )}
+          />
+        </KebabMenu>
+      </View>
+      {isOnline
```

Comment:
Is this necessary? In the native Android app the maps SDK has cached tiles so the maps sometimes work while offline.